release-team/role-handbooks/enhancements/README.md

```diff
@@ -139,13 +139,13 @@ It is important that this process be followed and documentation remain up-to-dat
+    - Enable the test-infra job that syncs enhancements to the GitHub project board based on the `lead-opted-in` label. Update the name of the [periodic-sync-enhancements-github-project](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml#L2) cronjob with the version number of the release and enable the cronjob by commenting the impossible cron and uncommenting the `interval`. You can see how this was done for the v1.29 release cycle in [this commit](https://github.com/kubernetes/test-infra/pull/30528/files#diff-9d86ca0a46a2f74a2cf59fff3d18cbba57b5b3489ecc00c36b03f6b6a0c2ac3a). Make sure you remove the `lead-opted-in` label from all the KEPs of the previous release before enabling the job. Otherwise KEPs from the previous cycle will be populated in the board.
 - Create a shortlink for the Tracking Board
   - Create a free account on [bitly](https://bitly.com/) to create a shortlink for the new Enhancement Tracking Board following the pattern `k8sxyy-enhancements`, e.g. <https://bit.ly/k8s127-enhancements>.
 - Make a [pull request](https://github.com/kubernetes/sig-release/pull/1411) to add the shortlinked Enhancement Tracking Board to the current release page in [sig-release][sig-release].
 - Make a pull request to add the shortlinked Enhancement Tracking Board to [kubernetes/enhancements README page](https://github.com/kubernetes/enhancements#enhancements-tracking-board).
 - Find [Issues][enhancements-issues] from previous milestone that have graduated to Stable. Check to see if the KEP status has been updated to `implemented`. If it has, close the issue. If it has not, ask the issue contact to both update the KEP status field and close the Enhancement issue once the update PR has merged.
-- Find [Issues labeled `lead-opted-in`](https://github.com/kubernetes/enhancements/issues?q=+is%3Aissue+label%3Alead-opted-in+) and remove the `lead-opted-in` label from all issues. Enhancements must be explicitly opted into each release.
+- Find [Issues labeled `lead-opted-in`](https://github.com/kubernetes/enhancements/issues?q=+is%3Aissue+label%3Alead-opted-in+) and remove the `lead-opted-in` label from all issues. Enhancements must be explicitly opted into each release. Make sure you remove the `lead-opted-in` labels from all the KEPs of the previous release before you turn on the test-infra job. Else KEPs from the previous release will be added to the board.
```

Comment:
Actually, let's move this item on top of the instruction to update the test-infra job which updates the automation for board sync?

Reply:
Hi @sreeram-venkitesh, circling back on this one

![image](https://github.com/user-attachments/assets/a4a27a6f-6063-44dc-ab78-a470bdc60024)


Let's move the first "Make sure you remove the `lead-opted-in` ..." instruction to a separate line on its own & use some emphasis (bold/italicize/quotes) to catch the eye.

Reply:
How does this look? I realized the warning is repeated twice, but I thought of keeping it for good measure.

![image](https://github.com/user-attachments/assets/e8a0dcf1-5e56-4510-aa24-cb156b1f2b32)